### ISSUE TITLE
Evaluation Questions

### ISSUE THREAD

**Issue by YBYBZhang** [luogen1996/Real-time-Global-Inference-Network#1]
Hi, luogen. I am reproducing the results on referit datasets. 
I download the pretrained model from [this link](https://1drv.ms/u/s!AmrFUyZ_lDVGgQErMsd5oJx-S9x-?e=dnjqMK) and run `python code/main_dist.py $exp_name --ds_to_use='refclef' --resume=True --only_val=True --only_test=True`.  
However, the validation accuracy is not so good, only 23.91%.  My configuration is the same as the configs/cfg.json. Maybe I ignore something important. 
I'm looking forward your help. Thank you so much!
